src/applications/pact-act/utilities/results-1-1-dynamic-content.js

```diff
@@ -27,20 +27,22 @@ export const getDynamicContent = formResponses => {
   for (const shortName of answeredQuestions) {
     const formAnswer = formResponses[shortName];
 
-    for (const dataForOneQuestion of dynamicContent[shortName]) {
-      // For checkbox answers
-      const dynamicContentMatchesArrayAnswer =
-        Array.isArray(formAnswer) &&
-        formAnswer.includes(dataForOneQuestion.response);
-
-      const dynamicContentMatchesStringAnswer =
-        formAnswer === dataForOneQuestion.response;
-
-      if (
-        dynamicContentMatchesArrayAnswer ||
-        dynamicContentMatchesStringAnswer
-      ) {
-        dynamicContentMatchingAnswer.push(dataForOneQuestion.content);
+    if (dynamicContent?.[shortName]) {
```

Comment:
Not every path that leads to Results pages 1.1-1.2 has dynamic content anymore (since we added a new Burn Pit question), so the dynamic content function handles the cases where we don't.